an-apluss/AutoMart
# #166273089 Write test suite for car endpoints

#### What does this PR do?
- Write test suite for car endpoints

#### Description of Task to be completed?
- write test cases for `POST /api/v1/car` endpoint
- move image to tests directory as dummy image for test
- write test cases for `PATCH /api/v1/car/:carId/status` endpoint
- write test cases for `PATCH /api/v1/car/:carId/price` endpoint
- write test cases for `GET /api/v1/car/:carId` endpoint
- write test cases for `GET /api/v1/car?status=available` endpoint
- write test cases for `GET /api/v1/car?status=available&min_price=XXXValue&max_price=XXXValue` endpoint
- write test cases for `DELETE /api/v1/car/carId` endpoint
- write test cases for `GET /api/v1/car` endpoint
- create dummy data for car
- delivers[#166273089]

#### How should this be manually tested?
- Clone the repository using `git clone https://github.com/an-apluss/AutoMart.git` . in your terminal
- run `npm install` on your terminal
- run `npm test` to see test cases that fail

#### Any background context you want to provide?
Create environment variables `JWTPRIVATEKEY= yourjwtprivatekey` 

#### What are the relevant pivotal tracker stories?
- PT Story link - ( [166273089](https://www.pivotaltracker.com/story/show/166273089) )

#### Screenshots(if appropriate)
-N/A
